napari/utils/interactions.py

```diff
@@ -41,6 +41,9 @@ def hello_world(layer, event):
     event : Event
         Mouse event
     """
+    if hasattr(obj, 'visible') and not obj.visible:
```

Comment:
```suggestion
    if not getattr(obj, 'visible', False):
```

Why do we expect that attribute may not be present?

Reply:
Checking the functions docstring seems like the `obj` could be of type `ViewerModel` or `Layer` I think `ViewerModel` doesn't have a visible attribute so that's why I added the validation, but maybe `ViewerModel` has such an attribute or these functions aren't being used passing as `obj` a `ViewerModel` instance? 🤔

Reply:
So maybe we should use `isinstance(obj, ViewerModel)`? To prevent the silence of real warning.

Reply:
I think using `isinstance` for the validation makes sense :+1:, but then to only check for the `visible` attribute when appropriate the validation should end up being something like:

```python
if isinstance(obj, Layer) and not obj.visible:
    return
```
right? or what do you think?

Reply:
> To prevent the silence of real warning.

what do you mean here @Czaki?